.github/workflows/install.yaml

```
@@ -14,17 +14,17 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install Poetry
-        uses: snok/install-poetry@v1
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
 
       - name: Create wheel and tar.gz
         run: |
-          poetry build
+          uv build
 
       - name: Install package
         run: |
-          python -m pip install dist/*.whl
+          uv run python -m pip install dist/*.whl
```

Oh actually, I think this is failing because you can do 

```suggestion
          uv pip install dist/*.whl
```

---

Ahh, thanks.